src/corona_hakab_model/generation/circles_consts.py

```diff
@@ -22,21 +22,25 @@ class CirclesConsts(NamedTuple):
             "ages": [10, 40, 70],
             "age_prob": [0.30, 0.45, 0.25],
             "teachers_workforce_ratio": 0.04,  # ratio of teachers out of workforce
+            "kindergarten_workforce_ratio": 0.002,  # ratio of kindergarten workers out of workforce
             "connection_type_prob_by_age_index": [
                 {
                     ConnectionTypes.Work: 0,
-                    ConnectionTypes.School: 0.95,
+                    ConnectionTypes.Kindergarten: 0.1,
+                    ConnectionTypes.School: 0.85,
                     ConnectionTypes.Family: 1.0,
                     ConnectionTypes.Other: 1.0,
                 },
                 {
                     ConnectionTypes.Work: 0.9,
+                    ConnectionTypes.Kindergarten: 0,
```

Comment:
What about kindergarten teachers?

Reply:
If I understand the question correctly, it is the same as with schools. The connection type for kindergarten workers is ConnectionTypes.Kindergarten